.ci/macos-steps.yml

```diff
@@ -17,7 +17,7 @@ steps:
 
 - bash: |
     brew update
-    brew install ccache pkg-config arpack eigen glpk hdf5 json-c lapack lzo nlopt snappy xz
+    brew install ccache pkg-config arpack eigen glpk hdf5 json-c lapack lzo nlopt snappy xz libomp
```

Comment:
even though this is going to /usr/local ...

but I am not sure, just a thought,  I am not a mac user